features/PJS002_Project-declaration.feature

```diff
@@ -0,0 +1,22 @@
+@implementer-agreement
+@PJS
+@BLT
+@version1
+@E00010
+Feature: PJS002 - Project Declaration
```

Comment:
```suggestion
Feature: PJS002 - Correct elements related to project
```
Please adjust the file name accordingly

Reply:
https://github.com/buildingSMART/ifc-gherkin-rules/pull/204/commits/6b511ea9a6976d7cf0d1aaa77eb1863fa0d37808